runner/log/journald.go

```diff
@@ -40,7 +43,7 @@ func (j Journald) ID() string {
 // Run attempts to pull logs from journald via shell command, e.g.:
 // journalctl -x -u {name} --since '3 days ago' --no-pager > {destDir}/journald-{name}.log
 func (j Journald) Run() op.Op {
-	o := runner.NewSheller("journalctl --version", nil).Run()
+	o := runner.NewSheller("journalctl --version", j.Redactions).Run()
```

Comment:
I was a bit nervous about redacting intermediate runners that aren't in the last Op that get returned, but I see now that ofc we do return the results when it errors. We might as well let the runner redact itself like we have here - it's unlikely to cause issues in the current log runners cases.

Reply:
Yeah, at first I thought the same thing and planned on writing an Op-redacter that we could just use a whole bunch (`redact.Op(op.Op)` or perhaps the same thing in the Op package). Thankfully this simple implementation seems fine for now.